lib/shopify/resources/customer.ex

```diff
@@ -63,4 +65,34 @@ defmodule Shopify.Customer do
 
   @doc false
   def search_url, do: @plural <> "/search.json"
+
+  @doc false
+  def invite_url(id), do: @plural <> "/#{id}/send_invite.json"
+
+  @doc false
+  def orders(session, id, params \\ %{}) do
+    session
+    |> Request.new(@plural <> "/#{id}/orders.json", params, Order.plural_resource())
+    |> Client.get()
+  end
+
+  @doc false
+  def account_activation_url(session, id) do
+    session
+    |> Request.new(@plural <> "/#{id}/account_activation_url.json", %{})
+    |> Client.post()
+  end
+
+  @doc false
+  def send_invite(session, id) do
```

Comment:
Same as with custommer orders I guess 👍 

It just seems cleaner to have the main functions in their respective modules and use those functions for convenience in other resource modules.

Reply:
Here actually there is no endpoint for customer invites, the resource is just a convenience for generating the necessary JSON for the customized invite sending ... I don't think there's a need to create a nested resource here